### DISCUSSION TITLE
Stuck at: Your site is ready to be published

### DISCUSSION THREAD

**Discussion by lishap** [community/community#22023] [Other Feature Feedback, Questions, & Ideas]
<p>For some reason, my Pages tab is stuck at “Your site is ready to be published at…”. But why can’t I publish it? I’ve never had this issue with GitHub pages before.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/d/6da24d975cabcda26b7399c39ca2e37f1e0417d8.png" data-download-href="/uploads/short-url/fDRMaY903CbkTGqGPsWL7pYxBTq.png?dl=1" title="Screen Shot 2022-03-28 at 2.32.15 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/33298347/181068307-1fd3d1b1-c062-4b86-a8d8-0cfd6a878448.png" alt="Screen Shot 2022-03-28 at 2.32.15 PM" data-base62-sha1="fDRMaY903CbkTGqGPsWL7pYxBTq" width="690" height="429" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/d/6da24d975cabcda26b7399c39ca2e37f1e0417d8_2_690x429.png, https://aws1.discourse-cdn.com/github/optimized/3X/6/d/6da24d975cabcda26b7399c39ca2e37f1e0417d8_2_1035x643.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/6/d/6da24d975cabcda26b7399c39ca2e37f1e0417d8_2_1380x858.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/d/6da24d975cabcda26b7399c39ca2e37f1e0417d8_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screen Shot 2022-03-28 at 2.32.15 PM</span><span class="informations">1778×1106 106 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by lishap** [comment, marked as answer]
<p>I figured it out! I had to go to the actions tab and re-run the failed deployment, seemed to be a server-side issue.</p>

**Comment by jfullstackdev** [comment]
<p>yup that’s correct, you need to re-run it</p>
